modules/integration/tests-integration/lazy-loading-test/src/test/java/org/wso2/appserver/integration/lazy/loading/util/TenantStatus.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Seems like you have an identical TenantStatus.java class in both these locations tests-common & tests-integration (utils)
Is there any specific requirement for duplicating this class?